packages/emptystate/src/react/__stories__/index.story.tsx

```diff
@@ -1,15 +1,15 @@
 import { storiesOf } from '@storybook/react'
-import PropTypes from 'prop-types'
-import React from 'react'
+import * as React from 'react'
```

Comment:
😢

Reply:
because the `*` is going to confuse me 😆

Reply:
Oh I think I might be able to change that back. I think I made that change before turning on the `esModuleInterop` flag.